src/Microsoft.ML.StandardLearners/Standard/Online/AveragedPerceptron.cs

```diff
@@ -37,6 +37,17 @@ public sealed class AveragedPerceptronTrainer :
         internal const string UserNameValue = "Averaged Perceptron";
         internal const string ShortName = "ap";
         internal const string Summary = "Perceptron is a binary classification algorithm that makes its predictions based on a linear function.";
+        internal const string DetailedSummary = @"Perceptron is a classification algorithm that makes its predictions based on a linear function.
```

Comment:
>Perceptron [](start = 50, length = 10)

I wish there was some way to author them in markdown files...  I know attributes require constants, but is there some way to do this?  We have done this for Py and R.

Reply:
The documentation system will support ingesting md files, in the future, and those will go away. 
Didn't want to duplicate what they are doing, but at the same time can't leave the docs with just one liners.